src/Common/Infrastructure/Controllers/ApiController.cs

```diff
@@ -143,6 +146,7 @@ public abstract class ApiController<T> : PublicApiController
                     DateTimeOffset.UtcNow,
                     responseContentType,
                     false,
+                    response.HeadersToKeyValuePairs(),
```

Comment:
Does this mean that all headers will be forwarded from the internal registers?

Reply:
No, it makes the headers available in `BackendResponse` so they can be forwarded if chosen (see [here](https://github.com/Informatievlaanderen/public-api/pull/264/files/3a9d6b3b40a601dab7da57483a493ff1f9dfe154#diff-f898cd6b53bc3546753d5689292080c0b0d3bce254be0e2884c023e004139dc4R29))